coverage-text: do not use colors if --colors=never

gkralik opened the pull request:
Don't use colors when using `--coverage-text` along with `--colors=never`.

See #1771 and #1892 

(replaces #1906)

gkralik committed:
add test for #1892 (--colors=never with --coverage-text)

if --colors=never is used with --coverage-text, no colored
output should be printed.

refs #1771
refs #1892

gkralik committed:
do not use colors in coverage-text when --colors=never

refs #1771
refs #1892

GrahamCampbell commented on src/TextUI/TestRunner.php:
```diff
@@ -490,6 +490,9 @@ class_exists($arguments['printer'], false)) {
                 if ($arguments['coverageText'] == 'php://stdout') {
                     $outputStream = $this->printer;
                     $colors       = $arguments['colors'];
+                    if($colors == PHPUnit_TextUI_ResultPrinter::COLOR_NEVER) {
```
why not just `$colors       = $arguments['colors'] && !PHPUnit_TextUI_ResultPrinter::COLOR_NEVER`?

GrahamCampbell replied:
just thinking about avoiding a double assignment - maybe your way is more readable though :)

gkralik replied:
yeah, just thought about the same thing... will change it

gkralik committed:
avoid double assignment

gkralik committed:
skip the coverage-text --colors=never test on PHP 7

there is no coverage driver for PHP 7 on travis right now,
so just skip it. it can be enabled as soon as xdebug supports
code coverage on PHP 7

GrahamCampbell commented:
:+1:

sebastianbergmann commented:
Merged manually, thanks.

GrahamCampbell commented:
:+1:
